releasenotes/notes/0.11/prepare-0.11-af688e532712c830.yaml

```diff
@@ -0,0 +1,66 @@
+---
+prelude: |
+  This release includes many new features and bug fixes. The highlights of this
+  release are more methods to improve working with edges in both the
+  :class:`~retworkx.PyGraph` and :class:`~retworkx.PyDiGraph` classes,
+  several new graph generators, and a new set of interactive traversal
+  functions :func:`~retworkx.bfs_search`, :func:`~retworkx.dfs_search`,
+  :func:`~retworkx.dijkstra_search`, and :class:`~retworkx.TopologicalSorter`
+  which enable iteratively interacting with the graph during different types
+  of traversal.
+
+  This release also introduces a new separate Rust crate ``retworkx-core`` which
```

Comment:
Do you want to link to `retworkx-core` on crates.io or wherever it's made public?

Reply:
Yeah that's probably not a bad idea. The link will be: https://crates.io/crates/retworkx-core but there is a bit of a race condition because I won't publish the crate until tagging but since the url is stable it'll resolve once it's there I'll add it now.

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/549/commits/b16f1549d0297a7459cd5332e19b97e316d7b99f